### ISSUE TITLE
Get pre-trained weights

### ISSUE THREAD

**Issue by TomHsu-MIddle** [WZMIAOMIAO/deep-learning-for-image-processing#318]
Can I get your pre-trained weights on GoogleDrive? I can't download it on your website. Please~

**Comment by TomHsu-MIddle** [comment]
BTW, I want shufflev2.

**Comment by WZMIAOMIAO** [comment]
sorry, Nope. 

if you want to get wights for tensorflow2, you can try:
- download weight for pytorch, https://download.pytorch.org/models/shufflenetv2_x1-5666bf0f80.pth
- rename ```shufflenetv2_x1-5666bf0f80.pth``` to ```shufflenetv2_x1.pth```
- using ```trans_weights.py``` convert ```shufflenetv2_x1.pth``` to ```shufflenetv2_x1_0.h5```
